draft: true (#891)
observablehq/framework: docs/markdown.md

---
```

The front matter supports the following options:

> - **title** — the title, defaults to the (first) first-level heading of the page, if any

Reviewer: ```suggestion
- **title** — the page title; defaults to the (first) first-level heading of the page, if any
```

- **toc** — if false, disables the [table of contents](./config#toc)

> - **index** — if [search](./search) is enabled, should this page be indexed (defaults to true for pages listed in the sidebar, and false for any other page)

Reviewer: ```suggestion
- **index** — whether to index this page if [search](./search) is enabled; defaults to true for listed pages
```

> - **draft** — whether this page is a draft; drafts are rendered in preview but skipped during build and deploy; drafts are not included in the default sidebar.

Reviewer: ```suggestion
- **draft** — whether to skip this page during build; drafts are also not listed in the default sidebar
```
